types/react-native/test/index.tsx

```diff
@@ -112,7 +112,7 @@ BackHandler.addEventListener("hardwareBackPress", () => {}).remove();
 
 BackAndroid.addEventListener("hardwareBackPress", () => {});
 
-interface LocalStyles {
+type LocalStyles = {
```

Comment:
I'm not 100% sure about this change but without it. it'll complaint about `LocalStyles` can not do string index?

Reply:
Yeah, the compiler assumes that `interface LocalStyles` might be subclassed and of course `interface SubLocalStyle extends LocalStyles` has no constraint that requires its properties to be of any particular type.

`type LocalStyles`, on the other hand, can't be subclassed, so it's fine to check its properties for assignability to a string index.